Changelog.md

```diff
@@ -1,26 +1,12 @@
 # Changelog
 
-## Unreleased new version
+## 0.4.2 'Bonjour des Pyrénées'
 
-Released yyyy-mm-dd.
+Released 2025-06-18.
 
-
-Breaking changes
-
-
-New
-
-
-Bug fixes
-
-
-Known issues
-
-
-Other changes
-
-
-Acknowledgements
+With this release, Rotonda switches to the latest version of the `roto` library.
+While many things have improved behind the scenes and under the hood, there are
+no breaking or otherwise noticeable changes for Rotonda users.
```

Comment:
The assignment operator is a new feature, but that's not super important. You could link to the Roto changelog I suppose.

Reply:
Added a link to the changelog.
As we do not use the assignment operator (yet) in any of the packaged examples, let's leave it for another time, if only to prevent confusion.